todo_app/src/comments.js

```diff
@@ -0,0 +1,42 @@
+export const comments = [
```

Comment:
Ah, you decided to use a `.js` file instead of `.json`. That's fine, but normally when a file only exports a single thing, and/or has a single purpose, we would use a `default` export. However what you've done here is technically still viable/correct.